commons/connector/upload/altoida/altoida_summary.avsc

```diff
@@ -10,7 +10,7 @@
     {"name": "age", "type": "int", "doc": "Age of the participant in years."},
     {"name": "yearsOfEducation", "type": "int", "doc": "Years of education in years."},
     {"name": "gender", "type": "int", "doc": "Gender of the participant 0=male, 1=female, 2=other."},
```

Comment:
I'm wondering if these and the following should not be enums for clarity?

Reply:
True. I will look into it.